Cargo.toml

```diff
@@ -10,54 +10,53 @@ readme = "README.md"
 [dependencies]
 actix = "0.7.9"
 actix-web = { version = "0.7.19", features = ["tls"], default-features = false }
-anyhow = "1.0.32"
+anyhow = "1.0.38"
 apple-crash-report-parser = "0.4.2"
-backtrace = "0.3.53"
-base64 = "0.11.0"
+backtrace = "0.3.55"
+base64 = "0.13.0"
 bytes = { version = "0.4.12", features = ["serde"] }
 cadence = "0.18.0"
 chrono = { version = "0.4.19", features = ["serde"] }
-console = "0.10.0"
-crc32fast = "1.2.0"
-derive_more = "0.15.0"
+console = "0.14.0"
+crc32fast = "1.2.1"
 env_logger = "0.7.1"
-failure = "0.1.6"
-flate2 = "1.0.14"
+failure = "0.1.8"
+flate2 = "1.0.19"
 fragile = "1.0.0" # used for vendoring sentry-actix
-futures = { version = "0.3", features = ["compat"] }
+futures = { version = "0.3.12", features = ["compat"] }
 futures01 = { version = "0.1.29", package = "futures" }
 glob = "0.3.0"
-humantime-serde = "1.0.0"
-ipnetwork = "0.16.0"
+humantime-serde = "1.0.1"
+ipnetwork = "0.17.0"
 # needed for gcs, see https://github.com/Keats/jsonwebtoken/pull/89
 jsonwebtoken = { git = "https://github.com/Keats/jsonwebtoken", rev = "b8627260b2902a1ab4fdda83083be3e0b0fb9b7f" }
 lazy_static = "1.4.0"
-log = { version = "0.4.8", features = ["serde"] }
-lru = "0.4.3"
-num_cpus = "1.12.0"
-parking_lot = "0.10.0"
+log = { version = "0.4.13", features = ["serde"] }
+lru = "0.6.3"
+num_cpus = "1.13.0"
+parking_lot = "0.11.1"
 pretty_env_logger = "0.4.0"
 procspawn = { version = "0.9.0", features = ["backtrace", "json"] }
-regex = "1.3.3"
+regex = "1.4.3"
 reqwest = { git = "https://github.com/jan-auer/reqwest", tag = "v0.11.0", features = ["gzip", "json", "stream", "trust-dns"] }
 rusoto_core = "0.46.0"
 rusoto_credential = "0.46.0"
 rusoto_s3 = "0.46.0"
 sentry = { version = "0.21.0", features = ["debug-images", "failure", "log"] }
```

Comment:
followup: update this and vendor the failure integration

Reply:
0.22.0 removed the failure integration, which is why I did not run this update. We use `failure` in two places: the Sentry actix middleware and the error handling middleware. Since I was looking into replacing `actix-web` with `warp`, we can likely get rid of that entirely.